task_manager/tasks/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from task_manager.tasks.views import (
+    TaskDetailsView,
+    TaskFormCreateView,
+    TaskFormDeleteView,
+    TaskFormEditView,
+    TaskListView,
+)
+
+urlpatterns = [
+    path("", TaskListView.as_view(), name="tasks_list"),
+    path("<int:id>/", TaskDetailsView.as_view(), name="task_details"),
+    path("create/", TaskFormCreateView.as_view(), name="create_task"),
+    path("<int:id>/edit/", TaskFormEditView.as_view(), name="update_task"),
```

Comment:
this url is hexlet app is `<int:id>/update/`

Reply:
same with other update endpoints (for other entities), sorry for not spotting it right away